src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person: { name, age, sex, isMarried, partnerName },
+}) => {
+  const personStatus = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
```

Comment:
with content on separate line code will be more readable
```suggestion
      <h2 className="Person__name">
        {`My name is ${name}`}
      </h2>
```